src/components/Products.jsx

```diff
@@ -14,6 +15,22 @@ const Products = () => {
     fetchProducts();
   }, []);
 
+  const addToOrder = async (id) => {
+    const toastSetting = { autoClose: 1500, toastId: "message-box" };
+    if (orderInProgress === false) {
+      const response = await axios.post("https://reqres.in/api/orders", {
+        params: { product_id: id },
+      });
+      setOrderInProgress(true);
+      toast(response.data.message, toastSetting);
+    } else {
+      const response = await axios.put("https://reqres.in/api/orders", {
```

Comment:
This is out of scope. the PUT request is used for updating the order. The question is if you are in control of what this code does?

Reply:
> This is out of scope. the PUT request is used for updating the order. The question is if you are in control of what this code does?

yes. I don't really understand this part 🫣. but I will learn and modify this today ( I just cope with old code and if no this part it will not work so even I am Confuse but feel works so I used it, and I know that's not right🫠🫢 )

Reply:
We've discussed the difference.